docs/API-functions.md

```diff
@@ -1256,6 +1256,7 @@ with the rest of the buffer next time to send it completely. In both
   * `SRT_ECONNLOST`: Socket `u` used for the operation has lost its connection.
   * `SRT_EINVALMSGAPI`: Incorrect API usage in **message mode**:
     * **live mode**: trying to send more bytes at once than `SRTO_PAYLOADSIZE`
+    or wrong source time was provided.
```

Comment:
or a nonzero time is provided in case when `SRTO_TSBPDMODE` is off - as you added that already.

Reply:
In the case you describe no error is returned. There was just a warning log message.

Reply:
Right.